packages/pluggableWidgets/color-picker-web/src/ColorPicker.tsx

```diff
@@ -0,0 +1,28 @@
+import { createElement, ReactNode, useCallback } from "react";
+import { executeAction } from "@mendix/piw-utils-internal";
+import { ColorPicker as ColorPickerComponent } from "./components/ColorPicker";
+import { ColorPickerContainerProps } from "../typings/ColorPickerProps";
+import "./ui/ColorPicker.scss";
+
+export function ColorPicker(props: ColorPickerContainerProps): ReactNode {
+    const { name, mode, tabIndex, type, onChange, colorAttribute, defaultColors, format, id, invalidFormatMessage } =
+        props;
+    const onChangeFn = useCallback(() => executeAction(onChange), [onChange]);
+    const onChangeComplete = useCallback(value => props.colorAttribute.setValue(value), [props.colorAttribute]);
+    return (
+        <ColorPickerComponent
+            id={id}
+            name={name}
+            color={colorAttribute.value as string}
```

Comment:
Are we sure this case is always valid? What if the user doesn't add a `colorAttribute` or the attribute has no value?

Reply:
`colorAttribute` is required so the user can't skip it, and it has no value on the case when the status still pending, but the value is undefined in that case, which is not breaking anything cause `input` can receive both, 
I guess I can add one more type here saying "undefined". What do you think?

Reply:
> but the value is undefined in that case, which is not breaking anything cause input can receive both

That's good. I would update the typing to include the undefined case (prob in the ColorPickerComponent and then remove this case), cuz right now we're saying that it always is a string while it could not be the case.